build-tests/install-test-workspace/workspace/common/pnpm-lock.yaml

```diff
@@ -5,27 +5,27 @@ importers:
   typescript-newest-test:
```

Comment:
Revert?

Reply:
This file updates every time I run `rush build`